docs/tools/cli/events.rst

```diff
@@ -0,0 +1,38 @@
+Command reference: ``avn events``
+==================================
+
+Here you’ll find the full list of commands for ``avn events``.
+
+
+List project events
+-------------------
+
+Commands for listing events related to a particular project.
+
+
+``avn events``
+'''''''''''''''''''''''
+
+Lists instance or integration creation, deletion or modification events.
```

Comment:
Is this like a log of events? It is not exactly clear what the events are? Should we maybe show more realistic info in the example?

Reply:
I had to look into this to answer the question and it turns out these are not logs (as I had thought) but more like an audit trail of operations performed on the project. I've updated this page to give more information and an output example